modules.json

```diff
@@ -228,38 +223,34 @@
                     },
                     "bam_sort_stats_samtools": {
                         "branch": "master",
-                        "git_sha": "f4596fe0bdc096cf53ec4497e83defdb3a94ff62",
-                        "installed_by": ["fastq_align_bowtie2", "fastq_align_bwa", "fastq_align_hisat2", "subworkflows"]
+                        "git_sha": "4352dbdb09ec40db71e9b172b97a01dcf5622c26",
+                        "installed_by": ["fastq_align_bowtie2", "fastq_align_bwa", "fastq_align_hisat2"]
                     },
                     "bam_stats_samtools": {
                         "branch": "master",
                         "git_sha": "f4596fe0bdc096cf53ec4497e83defdb3a94ff62",
-                        "installed_by": ["bam_dedup_stats_samtools_umitools", "bam_sort_stats_samtools", "subworkflows"]
+                        "installed_by": ["bam_dedup_stats_samtools_umitools", "bam_sort_stats_samtools"]
                     },
                     "fastq_align_bowtie2": {
                         "branch": "master",
-                        "git_sha": "cfd937a668919d948f6fcbf4218e79de50c2f36f",
+                        "git_sha": "0fe30831abbc2ed115e46e92330edf38f56edc3d",
                         "installed_by": ["subworkflows"]
                     },
                     "fastq_align_bwa": {
                         "branch": "master",
-                        "git_sha": "cfd937a668919d948f6fcbf4218e79de50c2f36f",
+                        "git_sha": "b97197968ac12dde2463fa54541f6350c46f2035",
                         "installed_by": ["subworkflows"]
                     },
                     "fastq_align_hisat2": {
                         "branch": "master",
-                        "git_sha": "cfd937a668919d948f6fcbf4218e79de50c2f36f",
+                        "git_sha": "701ae347c4508fba1b7d65262596f278b6a11cb6",
                         "installed_by": ["subworkflows"]
                     },
                     "homer/groseq": {
                         "branch": "master",
                         "git_sha": "575e1bc54b083fb15e7dd8b5fcc40bea60e8ce83",
                         "installed_by": ["subworkflows"]
-                    }
-                }
-            },
-            "subworkflows": {
-                "nf-core": {
+                    },
                     "utils_nextflow_pipeline": {
                         "branch": "master",
```

Comment:
The subworkflow entry `utils_nextflow_pipeline` is also duplicated. Can you please remove that as well?

Reply:
I'm not seeing where? Can you add a suggestion on the section your seeing?

Reply:
Forget what I said...I meticulously compared the SHAs and didn't notice that there are both, nextflow and nf-core utils.

```
                  "utils_nextflow_pipeline": {
                        "branch": "master",
                        "git_sha": "5caf7640a9ef1d18d765d55339be751bb0969dfa",
                        "installed_by": ["subworkflows"]
                    },
                    "utils_nfcore_pipeline": {
                        "branch": "master",
                        "git_sha": "5caf7640a9ef1d18d765d55339be751bb0969dfa",
                        "installed_by": ["subworkflows"]
                    },
```